repl.nix

```diff
@@ -6,6 +6,6 @@ in
 { inherit flake; }
 // flake
 // builtins
-// nixpkgs
+// flake.nixosConfigurations.${hostname}
 // nixpkgs.lib
-  // flake.nixosConfigurations.${hostname}
+  // nixpkgs
```

Comment:
There was an issue that `flake.nixosConfigurations.<hostname>.pkgs` were overwriting `nixpkgs` which was not intended